rust/private/test_runner_template.rs

```diff
@@ -0,0 +1,43 @@
+use std::collections::BTreeMap;
+use std::process::Command;
+
+#[cfg(target_family = "unix")]
+use std::os::unix::process::CommandExt;
+
+/// This is a templated function for defining a map of environment
+/// variables. The comment in this function is replaced by the
+/// definition of this map.
+fn environ() -> BTreeMap<&'static str, &'static str> {
+// {environ}
+}
+
+
+/// Simply replace the current process with our test
+#[cfg(target_family = "unix")]
+fn exec(environ: BTreeMap<&'static str, &'static str>) {
+    Command::new("{executable}")
```

Comment:
This should also take any args (which may be passed with things like the [`--test_arg`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_arg) flag) and pass them as extra `argv`.

Reply:
Added an `args` function